skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt

```diff
@@ -340,21 +351,24 @@ actual open class SkiaLayer internal constructor(
         }
     }
 
-    override fun setBounds(x: Int, y: Int, width: Int, height: Int) {
-        var roundedWidth = width
-        var roundedHeight = height
-        if (isInited) {
-            roundedWidth = roundSize(width)
-            roundedHeight = roundSize(height)
-        }
-        super.setBounds(x, y, roundedWidth, roundedHeight)
-        backedLayer.setSize(roundedWidth, roundedHeight)
+    override fun doLayout() {
+        val (roundedWidth, roundedHeight) = if (isInited) roundSize(width) to roundSize(height) else width to height
```

Comment:
Oh right, this was probably because the `contentScale` used to only be correct after `addNotify` was called. But that's not the case anymore. I'll remove the condition!